pkgdb/src/buildenv/assets/activate.d/set-prompt.fish

```diff
@@ -0,0 +1,22 @@
+if not set -q FLOX_PROMPT
+    set FLOX_PROMPT "flox"
+end
+
+if set -q NO_COLOR
+    set _flox "flox [$FLOX_PROMPT_ENVIRONMENTS]"
+else
+    set colorPrompt1 \e\[38\;5\;$FLOX_PROMPT_COLOR_1""m
+    set colorPrompt2 \e\[38\;5\;$FLOX_PROMPT_COLOR_2""m
+    set _floxPrompt1 $colorPrompt1$FLOX_PROMPT
+    set _floxPrompt2 $colorPrompt2"["$FLOX_PROMPT_ENVIRONMENTS"]"
+    set _flox (set_color --bold)$_floxPrompt1" "$_floxPrompt2
+end
+
+if not functions -q flox_save_fish_prompt
+    functions --copy fish_prompt flox_save_fish_prompt
```

Comment:
Nitpick, but I would call this `flox_saved_fish_prompt`, otherwise it sounds like a function that is _saving_ the prompt rather than just running the function `fish_prompt` that we've saved.

Reply:
Sounds good to me. Changing.